aws/ci_build

```diff
@@ -91,6 +95,7 @@ def main()
     commit_url = "https://github.com/code-dot-org/code-dot-org/commit/#{RakeUtils.git_revision}"
     # Notify the HipChat channels about what happened.
     projects = ARGV.join(' ')
+    projects = 'websites' if projects.empty?
```

Comment:
it could, but I think the alternative ends up being roughly just as complex so it's a bit of a wash.